Changelog.md

```diff
@@ -4,24 +4,44 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - ReleaseDate
+## [0.34.0] - ReleaseDate
```

Comment:
I'll bump this as soon as there's review-approval, and do the release straight after. Not sure if we can make it today (2021-12-20) but hopefully tomorrow at last.